### PR TITLE
Feature/oh 33 adding property details pages

### PR DESCRIPTION
**Pull request by darciegrace**
# Description

Property Details page created with Property Details Modal to show further information
App.js and routes updated

### CODE REVIEW

**Comment by darciegrace** [comment]
need view more button on marketplace to link property details

File: src/__tests__/unitTests/pages/Home/featureProperties.test.js [hunk 1]
```diff
@@ -3,7 +3,8 @@ import { shallow, configure } from "enzyme";
 import Adapter from "enzyme-adapter-react-16";
 import FeaturedProperties from "../../../../Pages/home/components/featuredProperties";
 import HouseItem from "../../../../Pages/home/components/houseItem.js";
-import { CardData } from "../../../../helpers/featuredPropetiesData/propertiesData";
+import CardData from "/src/helpers/featuredPropertiesData/buyPropertiesData.js";
```
Comment (gillo17) [inline src/__tests__/unitTests/pages/Home/featureProperties.test.js:6]: should be ../../../../helpers/featuredPropertiesData/buyPropertiesData.js

File: src/__tests__/ui-tests/propertyDetailsPage.test.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    assert.include(details, "Size (sq/Ft): 464");
+    assert.include(details, "Island: Island.maui");
+    assert.include(details, "Zipcode: 90210");
+    assert.include(details, "Move In Date: 2021/12/10");
+    assert.include(details, "Date Added: 2018/12/10");
+
+    // Check that the property images are displayed correctly
+    const images = await driver.findElements(By.css(".gallery img"));
+    assert.strictEqual(images.length, 3);
+    assert.include(await images[0].getAttribute("src"), "bed1.d93568add29c65b9f611.jpg");
```
Comment (gillo17) [inline src/__tests__/ui-tests/propertyDetailsPage.test.js:48]: This doesn't feel right, @kman8887 would u be able to take a look

File: src/Pages/propertyDetails/components/Styles/PropertyD.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
+/* Primary color */
```
Comment (gillo17) [inline src/Pages/propertyDetails/components/Styles/PropertyD.css:1]: Delete this its not used

File: src/Pages/propertyDetails/components/PropertyDetails.scss [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    padding: 20px;
+    border-radius: 6px;
+    box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+    margin-bottom: 20px;
+    font-size: 38px;
+    text-align: center;
+  }
+  
+  .propertyDetailsBlock > h3 {
+    color: #447399;
```
Comment (gillo17) [inline src/Pages/propertyDetails/components/PropertyDetails.scss:65]: Dont think there is enough contrast here between the text and background, Pa11y will probs shout at you

File: src/__tests__/ui-tests/propertyDetailsPage.test.js (hunk 2 above)
Comment (darciegrace) [reply src/__tests__/ui-tests/propertyDetailsPage.test.js:48]: thats the wrong img first of all my b

Comment (gillo17) [reply src/__tests__/ui-tests/propertyDetailsPage.test.js:48]: better ^

**Pull request merged**